aptos-move/aptos-gas/src/aptos_framework.rs

```diff
@@ -50,12 +51,17 @@ crate::natives::define_gas_parameters_for_natives!(GasParameters, "aptos_framewo
     [.ristretto255.point_sub, "ristretto255.point_sub", 1],
     [.ristretto255.point_parse_arg, "ristretto255.point_parse_arg", 1],
 
-
     // TODO(Alin): These SHA512 gas costs could be unified with the costs in our future SHA512 module
     // (assuming same implementation complexity, which might not be the case
+
+    // DEPRECATED
     [.ristretto255.sha512_per_byte, "ristretto255.scalar_sha512_per_byte", 1],
+    // DEPRECATED
     [.ristretto255.sha512_per_hash, "ristretto255.scalar_sha512_per_hash", 1],
 
+    [.ristretto255.sha2_512_per_byte, optional "ristretto255.sha2_512_per_byte", 1],
```

Comment:
We think we should eliminate the optional feature in the gas parameters. 

From our rollout methodology over features, we can assume that the gas schedule is not requested before it's been loaded.

Reply:
Will do.